### DISCUSSION TITLE
Javascript events with the new Blocks

### DISCUSSION THREAD

**Discussion by ChrisPlaneta** [woocommerce/woocommerce-gutenberg-products-block#11930] [Q&A - Extension Integrations and Customizations] [answered]
Hi

I've been looking for a list of Javascript events that get fired when certain actions withing Woo happen on the front end. It seems that the ones that were previously available with Woo no longer work. Can you please point me in the direction where I can find their list? So far I only found this one >>> wc-blocks_added_to_cart

Thank you!

**Comment by opr** [comment]
Hi @ChrisPlaneta - thanks for opening this discussion!

On Cart/Checkout we're now mostly using data stores to track the cart/checkout state, and don't really fire off any JS events when things happen, besides what [we have documented here](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/third-party-developers/extensibility/checkout-block/dom-events.md#wc-blocks_removed_from_cart)

As for other blocks I will ask someone from the team to give some input about that.

Could you also let us know what you are specifically trying to achieve? You'll likely need a different approach from what you would have used in the shortcodes, so knowing the desired end-result would let us help you better.

**Comment by opr** [comment, marked as answer]
Closing as https://github.com/woocommerce/woocommerce-blocks/issues/11977 provides more information.
